nornir_shared/plot.py

```diff
@@ -224,32 +224,31 @@ def PolyLine(PolyLineList, Title=None, XAxisLabel=None, YAxisLabel=None, OutputF
         ...
     '''
 
-    colors = ['black', 'blue', 'green', 'yellow', 'orange', 'red', 'purple']
```

Comment:
The way colors are assigned by line length unfortunately needs refactoring elsewhere before this can be checked in.  This existing color selection system is used for the drift settle report graphic.  I'm fine passing an array of colors per-line, but the other callers of PolyLine need to be updated.

Reply:
Also... I think if the caller is handling color then we can check "if 'color' in kwargs" if it is not in kwargs we can apply a default color scheme here.

Reply:
I'm going to implement per-line color assignment as an optional behavior specified by an enum argument to PolyLine(). Assigning by line length will be the default--let me know what you think of that approach once you see it.